src/modules/dashboard/components/ExpenditurePage/Stages/StakeExpenditureDialog.tsx

```diff
@@ -0,0 +1,169 @@
+            className={styles.title}
+          >
+            <FormattedMessage {...MSG.header} />
+            <div className={styles.forceContainer}>
+              <FormattedMessage {...MSG.force} />
+              {/* Connect to the form in next PR */}
+              <Form initialValues={{ force: false }} onSubmit={() => {}}>
+                <div className={styles.toggleContainer}>
+                  <Toggle name="force" appearance={{ theme: 'danger' }} />
+                </div>
+              </Form>
+              <Tooltip
+                content={
+                  <div className={styles.tooltip}>
+                    <FormattedMessage id="tooltip.forceAction" />
+                  </div>
+                }
+                trigger="hover"
+                placement="top-end"
+              >
+                <Icon name="question-mark" className={styles.questionIcon} />
+              </Tooltip>
+            </div>
+          </Heading>
+        </DialogSection>
+        <DialogSection appearance={{ border: 'bottom' }}>
+          <div className={styles.stakeContainer}>
+            <FormattedMessage {...MSG.stake} />{' '}
+            <div className={styles.label}>
+              <span className={styles.icon}>
+                <TokenIcon
+                  className={styles.tokenIcon}
+                  token={activeToken}
+                  name={activeToken.name || activeToken.address}
+                />
+              </span>
+
+              <Numeral
+                unit={getTokenDecimalsWithFallback(activeToken.decimals)}
+                value={
+                  activeToken.balances[COLONY_TOTAL_BALANCE_DOMAIN_ID].amount
+                }
+              />
+              <span className={styles.symbol}>{activeToken.symbol}</span>
+            </div>
+          </div>
+        </DialogSection>
+        <DialogSection>
+          <div className={styles.messageContainer}>
+            <p className={styles.messageParagraph}>
+              <FormattedMessage {...MSG.descriptionText} />
+            </p>
+            <p className={styles.messageParagraph}>
+              <FormattedMessage {...MSG.descriptionText2} />
+            </p>
+          </div>
+        </DialogSection>
+      </div>
+      <DialogSection appearance={{ align: 'right', theme: 'footer' }}>
+        <Button
+          appearance={{ theme: 'secondary', size: 'small' }}
+          onClick={() => cancel()}
+          text={MSG.cancelText}
+          style={{
+            color: styles.textColor,
+            fontWeight: 400,
+            fontSize: styles.sizeNormal,
+          }}
+        />
+        <Button
+          appearance={{
+            theme: 'primary',
+            size: 'large',
+          }}
+          autoFocus
+          onClick={handleSubmit}
+          text={MSG.confirmText}
+          style={{
+            fontWeight: 700,
+            width: styles.submitButtonWidth,
+            marginLeft: styles.buttonMarginLeft,
+          }}
```

Comment:
Same as above

Reply:
I wanted styles from the Button component so I didn't use className. I deleted unnecessary styles, now it looks like this:
` style={{
     width: styles.submitButtonWidth,
   }}`